Products/Zuul/facades/eventclassesfacade.py

```diff
@@ -163,6 +163,36 @@ def getTransformTree(self, uid):
         })
         return transforms
 
+    def getTransformState(self, uid):
+        """
+        Returns a transform state.
+        """
+        obj = self._getObject(uid)
+        return obj.transformEnabled
+
+    def changeTransformState(self, uid):
```

Comment:
API nitpick: this should be `setTransformEnabled` taking two arguments, the transform's UID and the boolean to set the flag to.  Having a method that toggles state should be avoided in general; it's better to set state explicitly because it's unambiguous and cleaner to read and test.